assets/reader-activation/auth.scss

```diff
@@ -196,6 +194,21 @@
 				overflow-y: hidden;
 			}
 		}
+		&__have-account {
+			position: absolute;
+			left: 0;
+			width: 100%;
+			top: calc( -1em - 23px );
+			background-color: var( --newspack-cta-color );
+			padding: 4px 15px;
+			margin-bottom: 11px;
+			padding-top: 6px !important;
+			padding-bottom: 8px !important;
+			&,
+			a {
+				color: var( --newspack-cta-contrast-color );
```

Comment:
This element still has its hover color hard-coded to `#111`, which doesn't look great with the default red background:

<img width="587" alt="Screen Shot 2022-09-21 at 2 21 16 PM" src="https://user-images.githubusercontent.com/2230142/191603254-5767f30a-abfb-4f29-91d7-e3b8d6ff9330.png">

Recommend a light grey or even just keeping it white.

Reply:
Fixed in b675cecf6d71696f371af7165b938f82d0596717